maxim/logger/litellm/tracer.py

```
@@ -96,13 +112,14 @@ def log_pre_api_call(self, model, messages, kwargs):
                 kwargs["optional_params"] if "optional_params" in kwargs else {}
             )
             request_messages: list[GenerationRequestMessage] = []
+            input_text = self._extract_input_from_messages(messages)
             for message in messages:
                 request_messages.append(
                     GenerationRequestMessage(
                         role=message.get("role", "user"),
                         content=message.get("content", ""),
                     )
-                )
+                )                
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)